onnx/version_converter/IntraDomainConverter.h

```diff
@@ -0,0 +1,212 @@
+// Converter for ONNX models between different opset versions
+// in the same domain.
+
+#pragma once
+
+#include "onnx/version_converter/BaseConverter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct IntraDomainVersionConverter : BaseVersionConverter {
```

Comment:
DefaultVersionConverter extends this, so it can't be final.  However, that one might be.  I haven't done so in case we want to be able to extend it (for example to implement more restrictions on the ops we convert/the process of doing so).

Reply:
ok makes sense, didn't see that DefaultVersionConverter inherits from it.